patches/tModLoader/Terraria/Main.cs.patch

```diff
@@ -3531,6 +3531,15 @@
  				Main.PlaySound(7, -1, -1, 1, 1f, 0f);
  			}
  		}
+@@ -37535,7 +_,7 @@
+ 				{
+ 					if (Main.item[i].active)
+ 					{
+-						Microsoft.Xna.Framework.Rectangle value = new Microsoft.Xna.Framework.Rectangle((int)((double)Main.item[i].position.X + (double)Main.item[i].width * 0.5 - (double)Main.itemTexture[Main.item[i].type].Width * 0.5), (int)(Main.item[i].position.Y + (float)Main.item[i].height - (float)Main.itemTexture[Main.item[i].type].Height), Main.itemTexture[Main.item[i].type].Width, Main.itemTexture[Main.item[i].type].Height);
++						Microsoft.Xna.Framework.Rectangle value = ItemLoader.ItemHoverHitbox(Main.item[i]);
```

Comment:
Our own method is potentially redundant if it is only used here. (can be considered sort of exasperate when redirecting to other classes when not needed) Oh well...